<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Размер крючка</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="bait">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Проесть еду</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="eat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Проесть наживку</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>13</number>
     </property>
    </widget>
    <widget class="QTextEdit" name="key_bait">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextEdit" name="key_eat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="key_bait2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Клавиша наживки</string>
     </property>
    </widget>
    <widget class="QLabel" name="key_eat2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Клавиша еды</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>150</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="2" column="1">
       <widget class="QPushButton" name="start">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="stop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="settings">
        <property name="text">
         <string>ConfirmSettings</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="tophint">
     <property name="geometry">
      <rect>
       <x>319</x>
       <y>10</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>tophint</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="wait_bait">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="wait_eat">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QKeySequenceEdit" name="keySequenceEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Задержка наживки</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Задержка еды</string>
     </property>
    </widget>
    <zorder>tophint</zorder>
    <zorder>label</zorder>
    <zorder>bait</zorder>
    <zorder>eat</zorder>
    <zorder>spinBox</zorder>
    <zorder>key_bait</zorder>
    <zorder>key_eat</zorder>
    <zorder>key_bait2</zorder>
    <zorder>key_eat2</zorder>
    <zorder>formLayoutWidget</zorder>
    <zorder>lineEdit</zorder>
    <zorder>wait_bait</zorder>
    <zorder>wait_eat</zorder>
    <zorder>keySequenceEdit</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBotFishing">
    <property name="title">
     <string>BotFishing</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGroup_Vk"/>
    <addaction name="actionTutorial_2"/>
   </widget>
   <addaction name="menuBotFishing"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGroup_Vk">
   <property name="text">
    <string>Group Vk</string>
   </property>
  </action>
  <action name="actionTutorial_2">
   <property name="icon">
    <iconset theme="https://www.twitch.tv/dark_elit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Tutorial</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
